specification/azsadmin/resource-manager/deployment/readme.md

```diff
@@ -72,10 +80,15 @@ input-file:
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/Deployment.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ActionPlan.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ActionPlanOperation.json
+  - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ActionPlanOperationAttempt.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/FileContainer.json
+  - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/Locations.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ProductDeployment.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ProductPackage.json
   - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/ProductSecret.json
+  - $(this-folder)/Microsoft.Deployment.Admin/preview/2019-01-01/OperationResults.json
+  - $(this-folder)/Microsoft.Deployment.Admin/preview/2018-07-01/FileContainer.json
```

Comment:
Product Package and File Container are supported only with 2018-07-01 api version?

Reply:
Product Package and File Container are supported both in '2018-07-01' and '2019-01-01' versions

Reply:
Lets include the 2019-01-01 version here then. Do we have any changes between those api versions for ProductPackage and FielContainer?

Reply:
I have already included 2019-01-01 and 2018-07-01 versions of FileContainer and ProductPackage in the 'read.md' file. Please review it . We don't have any changes for the both the versions . Few ARM services are still calling older versions of FileContainer and ProductPackages .

Reply:
Here the version should be the latest 2019-0-01 instead of 2018-07-01. you can keep the 2018 version in a older tag. in the input-file: lets use 2019-01-01. This tag is used for azure powershell autogeneration

Reply:
updated